tests/fsharpqa/Source/CodeGen/Structure/DoNotInlineX.fs

```diff
@@ -4,6 +4,8 @@ open System
 open System.Diagnostics
 open System.IO
 
+let longtime = 30000                    // longtime is 30 seconds
```

Comment:
To make this more explicit I'd prefer `TimeSpan.FromSeconds(30)` and `.WaitForExit(longtime.TotalMilliseconds)`.  Magic pre-computed numbers scare me.